Jamulus.pro

```diff
@@ -335,17 +345,20 @@ win32 {
 RCC_DIR = src/res
 RESOURCES += src/resources.qrc
 
-FORMS_GUI = src/clientdlgbase.ui \
-    src/serverdlgbase.ui \
-    src/clientsettingsdlgbase.ui \
-    src/chatdlgbase.ui \
-    src/connectdlgbase.ui \
-    src/aboutdlgbase.ui
+FORMS_GUI = src/aboutdlgbase.ui \
+    src/serverdlgbase.ui
+
+contains(CONFIG, "serveronly") {
+    message("server only, skipping client ui dialogs")
```

Comment:
Do we really need this?
We already have `Restricting build to server-only due to CONFIG+=serveronly.`in line 93 which should be sufficient for end-users?

```
$ qmake CONFIG=serveronly
Project MESSAGE: building version "3.8.2dev-bb7aaa2e" (intermediate in git repository)
Project MESSAGE: server only, skipping client ui dialogs
Project MESSAGE: server only, skipping client headers
Project MESSAGE: server only, skipping client ui headers
Project MESSAGE: server only, skipping client source
Project MESSAGE: server only, skipping client ui source
Project WARNING: CONFIG-=import_qpa_plugin is deprecated. Use QTPLUGIN.platforms=- instead.
```

Reply:
It was debugging - they can go, indeed.